src/javascripts/helpers/data/authData.js

```diff
@@ -0,0 +1,19 @@
+import firebase from 'firebase/app';
+import 'firebase/auth';
+
+const checkLoginStatus = () => {
+  firebase.auth().onAuthStateChanged((user) => {
+    if (user) {
+      // person is logged in
+      $('#auth').addClass('hide');
```

Comment:
You Probably did this to get auth working, but this is going to be hard to maintain in the long term if you’re hiding and showing classes for Dom elements

Reply:
^100% agree!

Reply:
Yeah I just did it to get auth working it will not stay this way. That would be a headache to write lol. Just wanted to clearly show auth works.